src/stories/foundation/colorDocs.tsx

```diff
@@ -0,0 +1,434 @@
+          <ul>
+            <li><strong>A:</strong> <em>Minimum</em> level. Text must have a contrast ratio of at least <b>3:1</b> for large text (18pt/14pt bold or larger) and <b>4.5:1</b> for normal text against its background. This level is the basic requirement for accessibility, ensuring that text is readable for most users with mild visual impairments.</li>
+            <li><strong>AA:</strong> <em>Mid-range</em> level. Text must have a contrast ratio of at least <b>4.5:1</b> for normal text and <b>3:1</b> for large text. This is the recommended level for most web content and is required for legal compliance in many regions (such as Section 508 in the US).</li>
+            <li><strong>AAA:</strong> <em>Highest</em> level. Text must have a contrast ratio of at least <b>7:1</b> for normal text and <b>4.5:1</b> for large text. This level is intended for content that needs to be accessible to the widest possible audience, including users with significant visual impairments. AAA is not required for most content, but is encouraged where possible.</li>
+          </ul>
+          <p style={{ fontSize: '13px', color: '#666', marginTop: '0.5em' }}>
+            <b>Source:</b> <a href="https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html" target="_blank" rel="noopener noreferrer">W3C: Understanding WCAG 2.1 Contrast (Minimum)</a>
+          </p>
+        </div>
+      </details>
+      <details>
+        <summary>Will this change with WCAG 3.0?</summary>
+        <div className="details-content">
+          <p>While WCAG 2.0 uses relative luminance to determine color contrast, WCAG 3.0 (potentially Dec 2025) is expected to use APCA (Accessible Perceptual Contrast Algorithm).
+            APCA is a new method for evaluating color contrast that takes into account human perception more effectively than previous methods.</p>
+          <p>The MyDataHelps color palette will be updated as necessary to comply with WCAG 3.0.</p>
+        </div>
+      </details>
+      <h3>Usage</h3>
+      <p><code>color: var(--ce-[hue]-[grade])</code> Example: <code>color: var(--ce-blue-40)</code> </p>
+      <h4>UI Use Cases</h4>
+      <ul>
+        <li>The 5 lightest (1-5) and darkest (95-99) grades are typically used as background colors.</li>
+        <li>The 55 grade is the lightest legible grade against backgrounds 1-5.</li>
+        <li>The 10 and 20 are typically used for border colors.</li>
+      </ul>
+      <h4>Data Visualization Use Cases</h4>
+      <ul className="data-viz-use-cases">
+        <li className="data-viz-air">Air Quality: Cyan</li>
+        <li className="data-viz-sleep">Sleep: Indigo</li>
+        <li className="data-viz-heart">Heart rate: Red-Orange</li>
+        <li className="data-viz-activity">Activity: Yellow</li>
+      </ul>
+      <div>
+        {colors.map((hue) => (
+          <ColorRow key={hue} hue={hue} />
+        ))}
+      </div>
+      <GradientSection />
+    </div>
+  );
```

Comment:
Build is failing and as a result, the deployed PR preview is out of date (still uses `var(--ce-grey-##)`)

Reply:
The build has a red X though:

Seems to be failing with this:

```
Unable to index ./src/stories/foundation/colorDocs.stories.mdx:
WARN   Error: Invariant failed: No matching indexer found for /home/runner/work/MyDataHelpsUI/MyDataHelpsUI/src/stories/foundation/colorDocs.stories.mdx
WARN     at invariant (/home/runner/work/MyDataHelpsUI/MyDataHelpsUI/node_modules/@storybook/core/dist/core-server/index.cjs:34564:11)
WARN     at ki.extractStories (/home/runner/work/MyDataHelpsUI/MyDataHelpsUI/node_modules/@storybook/core/dist/core-server/index.cjs:34764:5)
WARN     at /home/runner/work/MyDataHelpsUI/MyDataHelpsUI/node_modules/@storybook/core/dist/core-server/index.cjs:34709:53
```

Ignore what I said about var(--ce-grey), that was also related to the outdated diff.